src/OSmOSE/__init__.py

```diff
@@ -2,5 +2,7 @@
 from OSmOSE.timestamps import write_timestamp
 from OSmOSE.job import Job_builder
 from OSmOSE.Spectrogram import Spectrogram
+import OSmOSE.utils as utils
```

Comment:
why this importation ?

Reply:
utils was already in the __all__ list, so import is necessary. Otherwise relevance of utils in the __all__ list is not necessary in the Auxiliary code